src/v0/destinations/marketo_bulk_upload/util.js

```diff
@@ -336,6 +343,7 @@ const handleFileUploadResponse = (resp, successfulJobs, unsuccessfulJobs, reques
     resp.response?.result[0]?.importId
   ) {
     const { importId } = resp.response.result[0];
+    console.log(importId);
```

Comment:
this should done using logger with proper message including what is the destination id etc